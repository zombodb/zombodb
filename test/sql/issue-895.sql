select id, zdb.anyelement_cmpfunc(ctid, 'id:[1,4]'::zdbquery)
from so_posts
where so_posts ==> 'id:[1,2,3,4]'
order by id;

select id, zdb.anyelement_cmpfunc(so_posts, 'id:[1,4]'::zdbquery)
from so_posts
where so_posts ==> 'id:[1,2,3,4]'
order by id;