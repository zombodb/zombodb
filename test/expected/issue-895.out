select id, zdb.anyelement_cmpfunc(ctid, 'id:[1,4]'::zdbquery)
from so_posts
where so_posts ==> 'id:[1,2,3,4]'
order by id;
 id | anyelement_cmpfunc 
----+--------------------
  1 | t
  3 | f
  4 | t
(3 rows)

select id, zdb.anyelement_cmpfunc(so_posts, 'id:[1,4]'::zdbquery)
from so_posts
where so_posts ==> 'id:[1,2,3,4]'
order by id;
 id | anyelement_cmpfunc 
----+--------------------
  1 | f
  3 | f
  4 | f
(3 rows)

